modules/olmv1-adding-a-catalog.adoc

```diff
@@ -13,36 +13,34 @@ To add a catalog to a cluster, create a catalog custom resource (CR) and apply i
 
 . Create a catalog custom resource (CR), similar to the following example:
 +
-.Example `redhat-operators.yaml`
+.Example `my-redhat-operators.yaml` file
 [source,yaml,subs="attributes+"]
 ----
 apiVersion: catalogd.operatorframework.io/v1alpha1
 kind: ClusterCatalog
 metadata:
-  name: redhat-operators
+  name: my-redhat-operators
 spec:
   source:
     type: image
     image:
       ref: registry.redhat.io/redhat/redhat-operator-index:v{product-version} <1>
-      pullSecret: <pull_secret_name> <2>
-      pollInterval: <poll_interval_duration> <3>
+      pollInterval: <poll_interval_duration> <2>
```

Comment:
@michaelryanpeter I find pullSecret 
in https://docs.openshift.com/container-platform/4.17/extensions/catalogs/managing-catalogs.html, and better to change it as well.

Reply:
Some of the pull secret content was removed in another PR: https://github.com/openshift/openshift-docs/pull/85731/files#diff-b116ef93132bdba24282fe06afc198dcb30a9f23f1ee25f9f726b4bcdbbe60f5

If you look at the preview of this PR, you can see the mention of pull secrets have been removed: https://88209--ocpdocs-pr.netlify.app/openshift-enterprise/latest/extensions/catalogs/managing-catalogs

Reply:
ok, thanks